include/spine/query_factory.h

```diff
@@ -63,16 +70,132 @@ class QueryFactory
     template <typename T, typename = enable_param_if_protobuf_message<T>>
     QueryPtr all()
     {
-        return all(
+        return make_query(
                 T::descriptor()->file()->options().GetExtension(type_url_prefix),
                 T::descriptor()->full_name()
         );
     };
 
+    /**
+     * Creates a \b Query to read all entity states with the \b FieldMask
+     * applied to each of the results.
+     *
+     * Allows to set property paths for a \b FieldMask, applied to each of the query
+     * results. This processing is performed according to the
+     * [FieldMask specs](https://goo.gl/tW5wIU).
+     *
+     * In case the \b paths array contains entries inapplicable to the resulting entity
+     * (e.g. a \b path references a missing field), such invalid paths
+     * are silently ignored.
+     *
+     * @tparam T Protobuf Message type of a target entity.
```

Comment:
Not sure, but I think that several `@tparam`s should be inter-aligned (like in [Java code](https://github.com/SpineEventEngine/core-java/blob/master/server/src/main/java/io/spine/server/model/HandlerMethod.java#L222)) and don't have periods at the end.